src/calculateRentalCost.js

```diff
@@ -29,6 +29,20 @@
 
 function calculateRentalCost(days) {
   // write code here
+  const basePrice = 40;
+
+  const shortTerm = 3;
```

Comment:
Static constants (that is, those in which no calculations are made) are usually named with capital letters.
For example `SHORT_TERM`, `SHORT_TERM_DISCOUNT` etc.